services/batch-service.js

```diff
@@ -134,14 +134,18 @@ const asyncUpdateBatch = async (batchCollection, aBatch) => {
 }
 
 const createPrefix = (params, rootPath, orgID) => {
-    if (rootPath) {
-        return `${rootPath}/${params.type}`;
-    }
-    if (!orgID) {
+    if (!rootPath && !orgID) {
         throw new Error(ERROR.NEW_BATCH_NO_ORGANIZATION);
     }
-    const prefixArray = [orgID, params.submissionID];
-    prefixArray.push(params.type === BATCH.TYPE.METADATA ? BATCH.TYPE.METADATA : BATCH.TYPE.FILE);
+    const prefixArray = rootPath ? [rootPath, params.type] : [orgID, params.submissionID];
```

Comment:
I think we should throw and error if rootPath is null, undefined or empty string, because that should not happen under normal circumstances. So we don't need to pass in orgID and try to figure out a default rootPath anymore. In the reset of the function, we know for sure rootPath has valid values.